### PR TITLE
load contact from database before calling wasSeenRecently()

### PR DESCRIPTION
**Pull request by adbenitez**
this should solve the issue reported by @gerryfrancis in the testing group

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ConversationActivity.java [hunk 1]
```diff
@@ -1580,7 +1580,7 @@ public void handleEvent(@NonNull DcEvent event) {
                 || eventId == DcContext.DC_EVENT_MSG_DELIVERED)
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ConversationActivity.java:1580]: why update the state on `DC_EVENT_MSG_DELIVERED`? 

as this is not message coming from the recipient, the last-seen state should not been changed by the fact a message is sent out.

**Review by r10s — APPROVED** [review]

File: src/org/thoughtcrime/securesms/ConversationActivity.java (hunk 1 above)
Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationActivity.java:1580]: it is to update the state when you send a message to that contact and it no longer is "seen recently" (ex. to hide the dot after 10 min passed as opposed to the incoming messages that cause it to appear) it is not the ideal solution, but at least to reduce the chances to show the green dot forever (still happens if chat remains quiet without incoming/outgoing messages)

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationActivity.java:1580]: we have issues updating the last-seen indicator as well as message date label being "now" forever if there is not changes or incoming/outgoing messages and the same happens in the chat list, at some point would be nice to have some kind of timer to trigger a refresh if no refreshed after X time

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationActivity.java:1580]: perhaps there could be a core event when a contact's "seen recent" state changes :thinking:

Comment (r10s) [reply src/org/thoughtcrime/securesms/ConversationActivity.java:1580]: i see, then, for now, i'd just add a little comment to DC_EVENT_MSG_DELIVERED in this case.

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationActivity.java:1581]: added a comment in the point the listener register for handling this event

**Review by r10s — APPROVED** [review]

**Pull request merged**